src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import cx from 'classnames';
+
+import Person, { PersonProps } from 'Person/Person';
+
+type PersonPropsExtend = PersonProps & {
+  id: string;
+};
+
+export type PeopleListProps = {
+  people: PersonPropsExtend[];
+  variant?: 'grid' | 'list';
```

Comment:
Would using `layoutVariant` (instead of `variant`) be appropriate? It would match the name of the prop in the Person component, which I think is serving the same purpose.